src/SourceLocator/Ast/Exception/AstParse.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace BetterReflection\SourceLocator\Ast\Exception;
+
+use BetterReflection\SourceLocator\Located\LocatedSource;
+
+class AstParse extends \RuntimeException
```

Comment:
Also: AST is already AST, while parsing happens before AST. Some terminology confusion here, IMO.

Is this about parsing or is it about AST structure being invalid?